eng/testing/tests.browser.targets

```diff
@@ -91,7 +91,7 @@
     <WasmTestAppArgs Condition="'$(_XUnitBackgroundExec)' == 'true'">$(WasmTestAppArgs) -backgroundExec</WasmTestAppArgs>
```

Comment:
`-backgroundExec` is also triggered via MT, but uses different way how to pass the argument. Please unify it.

I wonder if the new MT runner is blocking `.Wait` on the UI thread now ? Could we check ?
Would the MT tests still pass without `-backgroundExec` ?

Reply:
`.Wait` will start throwing only with https://github.com/dotnet/runtime/pull/97052 
In your test it could be the underlying reason for many of `Failed to find loaded WebWorker`